katas/chicken-sexing/andres/chickenSexing.js

```diff
@@ -1,3 +1,11 @@
 export function correctness(bobsDecisions, expertDecisions) {
-
+    var score = 0;
+    for(var i=0; i<bobsDecisions.length; i++){
```

Comment:
Please add space between `for` and `(`, and for `<` operator. Also, please include space between `)` and `{`